apps/arena/lib/arena/entities.ex

```diff
@@ -65,7 +65,8 @@ defmodule Arena.Entities do
         bounties: [],
         selected_bounty: nil,
         bounty_completed: false,
-        current_basic_animation: 0
+        current_basic_animation: 0,
+        item_effect_duration: now
```

Comment:
Duration: now? What does this represent? Feels wrong to set a timestamp to a "duration".
Maybe rename this to `item_expires_at` if that's what it represents.

Reply:
it represent the time where the last item effect expires, i replaced it with `item_effect_expires_at`